permissions/plugin-configuration-as-code.yml

```diff
@@ -12,3 +12,7 @@ developers:
 security:
   contacts:
     jira: "jetersen"
+issues:
+  - github: *GH
+  - jira: 23170 # configuration-as-code-plugin
```

Comment:
Yep. See readme for explanation of the tradeoffs.

(Internally this queries Jira to determine ID from name because ID is needed for 'New Issue' URLs, so either works functionality wise.)